### PR TITLE
March 2023 Newsletter

### PR DESCRIPTION
**Pull request by ns-rse**
Feedback welcome :+1:

### CODE REVIEW

File: _newsletters/2023-03-30-newletter.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  career](https://github.blog/2023-03-06-unleash-your-potential-with-github-octernships-a-path-to-a-thriving-tech-career/) -
+  get paid to work on real-world projects whilst studying as part of GitHub's newly launched Octernships program.
++ [Guidelines for a healthy code review culture
+  MediaWiki](https://www.mediawiki.org/wiki/Guidelines_for_a_healthy_code_review_culture) - useful advice for developing
+  a code review culture in which feedback is welcomed without fear from [Mediawiki](https://www.mediawiki.org/).
++ [Why we need Research Software Engineers for Reproducible
+  Research](https://heidiseibold.ck.page/posts/why-we-need-research-software-engineers-for-reproducible-research) - Dr
+  Heidi Seibold on the utility of Research Software Engineers.
++ [PyTorch 2.0](https://pytorch.org/blog/pytorch-2.0-release/) - the next iteration of the popular open-source machine
+  learning framework is "faster, more Pythonic and Dynamic as ever_".
```
Comment (Robadob) [inline _newsletters/2023-03-30-newletter.md]: Spurious trailing underscore?

File: _newsletters/2023-03-30-newletter.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  Durham University, Department of Computer Science, Durham, UK, and online 20th April - 18th May 2023.
++ [RSE Midlands 2023 Annual Conference](https://rse-midlands.github.io/docs/event-24th-april-2023/) - hosted by the Digital
+  Research Service at the University of Nottingham's on Monday 24th April 2023.
++ [Teaching Reproducible Research and Open Science
+  Conference](https://www.sheffield.ac.uk/smi/events/teaching-reproducible-research-and-open-science-conference) - The
+  University of Sheffield, 20th-22nd June 2023.
++ [BigDat 2023 Summer – 7th International School of Big Data](https://bigdat.irdta.eu/2023su/) - Las Palmas de Gran
+  Canaria 17th - 21st July 2023. Online participation possible.
++ [RSE Conference 2023](https://rsecon23.society-rse.org/) - the UK Society of Research Software Engineering is holding
+  its 2023 conference in Swansea 5t-7th September 2023.
```
Comment (Robadob) [inline _newsletters/2023-03-30-newletter.md]: "5t"

(also the one above is the only one to have the date range hyphen delimited by spaces)

File: _newsletters/2023-03-30-newletter.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  your input on metrics for monitoring indicators of open research so have your say ([direct link to
+  survey](https://forms.office.com/pages/responsepage.aspx?id=MH_ksn3NTkql2rGM8aQVG83drixFredPnFQN6NWRAQ5UNjU1STFYMlNGVFJCWEMzWEs0V1ZVTzNQUyQlQCN0PWcu)).
++ [Research Software Engineers: Career Entry Points and Training Gaps](https://ieeexplore.ieee.org/document/10075674) -
+  for those considering or already working in Research Software Engineering this is well worth reading.
++ [Try Dark Mode on Desktop MATLAB with the ‘New Desktop for MATLAB’
+  beta](https://blogs.mathworks.com/matlab/2023/03/17/try-dark-mode-on-desktop-matlab-with-the-new-desktop-for-matlab/) -
+  Dark Mode is on its way to MATLAB.
+
+
+### Events
```
Comment (Robadob) [inline _newsletters/2023-03-30-newletter.md:56]: Be nice if events were organised chronologically SYCL at the bottom is April 23 so rather soon.

File: _newsletters/2023-03-30-newletter.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
++ [Research Software Engineering Sheffield - Unix Shell Fundamentals](https://rse.shef.ac.uk/training/workshop/2023-03-03-unix-shell)
+
+
+
+### Opportunities
+
+#### Funding
+
+
+#### Jobs
```
Comment (Robadob) [inline _newsletters/2023-03-30-newletter.md:106]: Closing dates?

File: _newsletters/2023-03-30-newletter.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+The Unix shell (which you may know as the terminal, console, or ‘bash’) has been around longer than most of its users have been alive. It has survived because it’s a powerful tool that allows users to perform complex and powerful tasks, often with just a few keystrokes or lines of code. It helps users automate repetitive tasks and easily combine smaller tasks into larger, more powerful workflows. Use of the shell is fundamental to a wide range of advanced computing tasks, including high-performance computing. This lesson will introduce you to this powerful tool.
+
++ [Research Software Engineering Sheffield - Unix Shell Fundamentals](https://rse.shef.ac.uk/training/workshop/2023-03-03-unix-shell)
+
+
+
+### Opportunities
+
+#### Funding
```
Comment (Robadob) [inline _newsletters/2023-03-30-newletter.md]: Probably comment out an empty section?

**Comment by ns-rse** [comment]
Good catches, thanks @Robadob :+1:

**Pull request merged**